multipass/how-to/get_started_with_multipas_linux.md

```diff
@@ -183,7 +183,7 @@ Memory usage:   170.4MiB out of 1.9GiB
 Mounts:         --
 ```
 
-We’ve created and deleted quite a few instances now. Let’s run multipass list to see what instances we currently have.
+We’ve created and deleted quite a few instances now. Let’s run a multipass list to see what instances we currently have.
```

Comment:
I think in this case "multipass list" is intended as a command, so it would also be OK to format it as code: 

> Let's run `multipass list` to see...